ci-operator/config/redhat-developer/openshift-jenkins-operator/redhat-developer-openshift-jenkins-operator-master.yaml

```diff
@@ -0,0 +1,65 @@
+
+base_images:
+  os:
+    name: ubi
+    namespace: ocp
+    tag: '7'
+  operator-registry:
+    name: "4.0"
+    namespace: ocp
+    tag: operator-registry
+
+resources:
+  '*':
+    limits:
+      memory: 6Gi
+    requests:
+      memory: 250Mi
+      cpu: 100m
+
+images:
+- from: os
+  inputs:
+    bin:
+      paths:
+      - destination_dir: .
+        source_path: /go/src/github.com/redhat-developer/openshift-jenkins-operator/out/operator 
+  dockerfile_path: openshift-ci/Dockerfile.deploy
+  to: openshift-jenkins-operator
+- from: operator-registry
+  dockerfile_path: openshift-ci/Dockerfile.registry.intermediate
+  to: operator-registry-base
+- from: operator-registry-base
+  dockerfile_path: openshift-ci/Dockerfile.registry.build
+  to: openshift-jenkins-operator-registry
+
+binary_build_commands: make build
+
+#build_root:
+#  project_image:
+#    dockerfile_path: openshift-ci/Dockerfile.tools
+
+tag_specification:
+  name: '4.1'
+  namespace: ocp
+
+tests:
+- as: lint
+  commands: VERBOSE=2 make lint
+  container:
+    from: src
+- as: unit
+  commands: VERBOSE=2 make test-unit-with-coverage upload-codecov-report
```

Comment:
i'm pretty sure this gets ignored.  "make unit" is what the generated job is going to invoke:

https://github.com/openshift/release/pull/4820/files#diff-c8b854b0ad264c2aaa9d0bc9fe756345R176

Reply:
> @akram i think you'd do well to try copying some of the existing OLM operator jobs (I assume this Jenkins operator will be an OLM operator).
> 
> see:
> https://github.com/openshift/release/tree/master/ci-operator/config/openshift/template-service-broker-operator
> 
> and
> https://github.com/openshift/release/tree/master/ci-operator/config/openshift/cluster-logging-operator

This is what I Tried to do @bparees but obviously some things are missing. I fixed a few, some other fixes are coming

Reply:
`make lint` now works for us. It was failing because of incorrect rule enforcement.